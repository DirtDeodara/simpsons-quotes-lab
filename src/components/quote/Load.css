@import url('https://fonts.googleapis.com/css?family=Bangers&display=swap');

.container {
  position: relative;
  text-align: center;
  font-size: 40px;
  margin-bottom: 20px;
}

.center {
  display: flex;
  justify-content: center;
}

.img {
  height: 40vh;
}

.h3 {
  font-family: 'Bangers', cursive;
  font-size: 3vw;
  font-weight: bolder;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(60%, -10%);
  margin-top: -1vw;
  color: #FFD90F;
  text-shadow: 2px 2px 0 black,
    2px -2px 0 black, -2px 2px 0 black,
    -2px -2px 0 black, 2px 0px 0 black,
    0px 2px 0 black, -2px 0px 0 black,
    0px -2px 0 black;
    color: #FFD90F;

 
}