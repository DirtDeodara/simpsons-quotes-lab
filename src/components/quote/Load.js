import React from 'react';
import PropTypes from 'prop-types';

function Load({ handleClick }) {
  return (
    <button onClick={handleClick}>Summon Forth</button>
  );
}

Load.propTypes = {
  handleClick: PropTypes.func.isRequired
};


export default Load;
