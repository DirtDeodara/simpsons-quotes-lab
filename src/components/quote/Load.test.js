import React from 'react';
import { shallow } from 'enzyme';
import Load from './Load';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<Load handleOnClick={() => { }}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
