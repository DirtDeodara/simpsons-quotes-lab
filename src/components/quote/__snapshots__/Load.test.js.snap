// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<section
  className="center"
>
  <section
    className="container"
  >
    <img
      className="img"
      src="/path/image"
      tabIndex="1"
    />
    <h3
      className="h3"
    >
      Mmmmore!
    </h3>
  </section>
</section>
`;
